franklin/builder/models.py

```diff
@@ -94,10 +207,20 @@ def build(self):
         self.save()
 
     def save(self, *args, **kwargs):
-        base_path = os.environ['BASE_PROJECT_PATH']
-        self.path = "{0}/{1}/{2}".format(base_path, self.owner, self.repo_name)
-        self.url = "{0}.{1}".format(self.repo_name, os.environ['BASE_URL'])
-        super(Site, self).save(*args, **kwargs)
-
+        if self.current_deploy:
+            if self.name == self.site.DEFAULT_ENV:
+                self.url = "{0}.{1}".format(self.site.name, 
+                                            os.environ['BASE_URL'])
```

Comment:
prod is just going to be `milagro.franklinstatic.com`. All other environments will have a `milagro-[env]`

Talking to Abrahams, part of this is simplicity and elegance. Log in with github, click to activate your project, push a single comment to your default branch, and BOOM!, you have a pretty-ish URL that is serviceable. We also talked about the idea of eventually having a setting to auto-detect and deploy ALL branches for a project. So if you want to share a feature with something, just push it to a remote branch and you'll have a sharable link for that. Might be overkill for what franklin is....but would be cool.